cif_ed.dic

```diff
@@ -0,0 +1,643 @@
+    loop_
+      _enumeration_set.state
+      _enumeration_set.detail
+         yes                      'Precession method is used.'
+         y                        'Abbreviation for "yes".'
+         no                       'Precession method is not used.'
+         n                        'Abbreviation for "no".'
+
+    _enumeration.default          no
+
+save_
+
+save_diffrn_measurement.integration
+
+    _definition.id                '_diffrn_measurement.integration'
+    _alias.definition_id          '_diffrn_measurement_integration'
+    _definition.update            2024-04-22
+    _description.text
+;
+    Code describing the technique used to sample as completely as possible
+    the accessible range of reciprocal space.
+;
+    _name.category_id             diffrn_measurement
+    _name.object_id               integration
+    _type.purpose                 Encode
+    _type.source                  Assigned
+    _type.container               Single
+    _type.contents                Code
+    loop_
+      _enumeration_set.state
+      _enumeration_set.detail
+         beam-tilt
+;
+         The radiation beam is tilted in order to collect intensities
+         in different regions of reciprocal space while the sample
+         remains stationary.
+;
+         continuous
+;
+         Synonym for "rotation" commonly used in 3D electron diffraction.
+;
+         rotation
+;
+         The detector records diffracted intensities continuously while the
+         sample is rotated. This is the usual mode for X-ray crystallography.
+;
+         stepwise
+;
+         The detector records diffracted intensities at intervals while the
+         sample is panned or tilted in discrete steps. Often used in 3D
+         electron diffraction.
+;
+
+    _enumeration.default          rotation
+
+save_
+
+save_diffrn_measurement.tracking
+
+    _definition.id                '_diffrn_measurement.tracking'
+    _alias.definition_id          '_diffrn_measurement_tracking'
+    _definition.update            2024-05-07
+    _description.text
+;
+    Yes or no flag indicating if a tracking method to follow the
+    crystal was used in the data acquisition. Typically used to track
+    very small crystals in electron diffraction experiments.
+;
+    _name.category_id             diffrn_measurement
+    _name.object_id               tracking
+    _type.purpose                 Encode
+    _type.source                  Assigned
+    _type.container               Single
+    _type.contents                Code
+    loop_
+      _enumeration_set.state
+      _enumeration_set.detail
+         yes                      'A crystal tracking method is used.'
+         y                        'Abbreviation for "yes".'
+         no                       'A crystal tracking method is not used.'
+         n                        'Abbreviation for "no".'
+
+    _enumeration.default          no
+
+save_
+
+save_diffrn_measurement.tracking_method
```

Comment:
The `_diffrn_measurement.tracking` and `_diffrn_measurement.tracking_method` seem to be quite tightly connected. This might lead to some contradictory datasets with `_diffrn_measurement.tracking`  set to `no`, but `_diffrn_measurement.tracking_method` still having a value.

Alternatively, the `_diffrn_measurement.tracking` item could be removed altogether with the logic being that the yes/no value can be determined from the presence of the `_diffrn_measurement.tracking_method` data item (inapplicable '.' value would imply that tracking was not used).